docs/wiki/User Guide/Configurations.md

```diff
@@ -137,7 +137,7 @@ The following configurations are inherited from the open source Kafka client con
 | default.replication.throttle                                      | Long	  | N         | null	                                                                                                                                                                                                                                                                                                                                                                                                                                                        | The replication throttle applied to replicas being moved, in bytes per second.	                                                                                                                                                                                                                                                                                             |
-| replica.movement.strategies                                       | List    | N         | [com.linkedin.kafka.cruisecontrol.executor.strategy.PostponeUrpReplicaMovementStrategy, com.linkedin.kafka.cruisecontrol.executor.strategy.PrioritizeLargeReplicaMovementStrategy, com.linkedin.kafka.cruisecontrol.executor.strategy.PrioritizeSmallReplicaMovementStrategy, com.linkedin.kafka.cruisecontrol.executor.strategy.PrioritizeMinIsrWithOfflineReplicasStrategy, com.linkedin.kafka.cruisecontrol.executor.strategy.BaseReplicaMovementStrategy]   | A list of supported strategies used to determine execution order for generated partition movement tasks.	                                                                                                                                                                                                                                                                     |
+| replica.movement.strategies                                       | List    | N         | [com.linkedin.kafka.cruisecontrol.executor.strategy.PostponeUrpReplicaMovementStrategy, com.linkedin.kafka.cruisecontrol.executor.strategy.PrioritizeLargeReplicaMovementStrategy, com.linkedin.kafka.cruisecontrol.executor.strategy.PrioritizeSmallReplicaMovementStrategy, com.linkedin.kafka.cruisecontrol.executor.strategy.PrioritizeMinIsrWithOfflineReplicasStrategy, com.linkedin.kafka.cruisecontrol.executor.strategy.PrioritizeOneAboveMinIsrWithOfflineReplicasStrategy, com.linkedin.kafka.cruisecontrol.executor.strategy.BaseReplicaMovementStrategy]   | A list of supported strategies used to determine execution order for generated partition movement tasks.                                                                                                                                                               |
```

Comment:
Nit: Can we update the spaces in the other rows to ensure that `|` is aligned for each column.